packages/xo-server-test/src/_xoConnection.js

```diff
@@ -111,7 +111,21 @@ class XoConnection extends Xo {
   }
 
   async createTempBackupNgJob(params) {
-    const job = await this.call('backupNg.createJob', params)
+    const job = await this.call(
+      'backupNg.createJob',
+      update(params, 'settings.', globalSetting => ({
```

Comment:
Is using `update` really necessary?

Also, do we want to change the `params` input?

Reply:
Using `update` is more simple and readable.

It's not have a side effect if we mutate the input in this cases because we all time pass a new object.

Reply:
Ok, let's use it, but avoid using it inline because it's not obvious this mutates `params`.

Reply:
Also, please document that `params` is mutated and that some default values are injected.

Reply:
Also, don't you think `defaultsDeep` would make more sense?

Reply:
Instead of `globalStetting`?

Reply:
No, `lodash.defaultDeeps` instead of `lodash.update`.

Reply:
Yes, `defaultDeeps`  is better.